README.md

```diff
@@ -1237,10 +1237,101 @@ class MyClass
 
 ### PhpDoc on methods
 
-We put PhpDoc comment on all methods with exception of constructors (can be skipped in some cases).
+#### No PhpDoc on fully strictly typed methods
 
-We put PhpDoc comment on constructors when IDE (for example PhpStorm) cannot guess classes of attributes,
-return type etc. It’s optional otherwise. For example, if we inject some scalar type, we must put PhpDoc comment.
+We put a PhpDoc comment on methods where the return and argument types cannot be determined by the IDE. For example, 
+prior to PHP 7.0, we would need a PhpDoc to type-hint scalar types. In PHP 7, the scalar types would be strictly 
+typed and PhpDoc is not needed.
+
+Wrong:
+```php
+/**
+ * @param Client $client
+ * @param string $type
+ *
+ * @return Money
+ */
+public function getLimit(Client $client, string $type): Money;
+
+public function setNumber($number);
+```
+
+Correct:
+```php
+public function getLimit(Client $client, string $type): Money;
+
+/**
+ * @param int $number
+ */
+public function setNumber($number);
+```
+
+#### PhpDoc on constructors
```

Comment:
I'm not sure this is needed, as it's already included by previous rule. I think we could just leave the examples?